src/gl/quadFeature.js

```diff
@@ -423,6 +423,7 @@ capabilities[quadFeature.capabilities.image] = true;
 capabilities[quadFeature.capabilities.imageCrop] = true;
 capabilities[quadFeature.capabilities.imageFixedScale] = false;
 capabilities[quadFeature.capabilities.imageFull] = true;
+capabilities[quadFeature.capabilities.canvas] = false;
```

Comment:
We don't need it, but I want to explicitly document what capabilities different renderers have rather than implicitly document them.